src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/RotatingStar.cpp

```diff
@@ -70,12 +75,39 @@ CstSolution::CstSolution(const std::string& filename,
       cst_file >> radius_[index] >> cos_theta_[index] >>
           rest_mass_density_[index] >> alpha_[index] >> rho_[index] >>
           gamma_[index] >> omega_[index] >> fluid_velocity_[index];
-      radius_[index] *= pow(equilibrium_kappa, polytropic_index_ * 0.5);
-      rest_mass_density_[index] *= pow(equilibrium_kappa, -polytropic_index_);
+      if (is_polytrope) {
```

Comment:
I think you're onto the right idea. Basically if the EoS in RotNS is set to `PT` (polytrope) then the normalization is done using the method in the 1992 paper, but if any other EoS is used, then it's done according to the method in the 1994 paper.
That being said, if the EoS is `PT`, you must use `is_polytrope=true`. The 1992 paper already uses geometric units, it just lets K (i.e. P=K\rho^\gamma) be a free parameter. RotNS assumes K=1 for its calculations. If the user doesn't want to rescale, they can just set `polytropic_constant=1.` and nothing will change. The 1994 method instead normalizes with CGS, so `is_polytrope=false` converts from CGS into geometric units, so the user would not want to use this for a `PT` EoS, even if they don't want to rescale.

(I specify setting the EoS to `PT` in RotNS because you could in principle import a polytrope as a tabulated EoS into RotNS, and it would use the 1994 normalization instead. But I don't know why someone would do this instead of the built-in polytrope functionality)

I'll still consider renaming the variable if you think it's still necessary, but personally I think it would make more sense to stay as is.

Reply:
Yeah I see, makes sense.  The main point is that the settings that are used should have agreement with the settings used in RotNS.  Can this text be added to replace the current doc for `CstSolution` in `RotatingStar.hpp`?

>  if the EoS in RotNS is set to PT (polytrope) then the normalization is done using the method in the 1992 paper, but if any other EoS is used, then it's done according to the method in the 1994 paper.
That being said, if the EoS is PT, you must use is_polytrope=true. The 1992 paper already uses geometric units, it just lets K (i.e. P=K\rho^\gamma) be a free parameter. RotNS assumes K=1 for its calculations. If the user doesn't want to rescale, they can just set polytropic_constant=1. and nothing will change. The 1994 method instead normalizes with CGS, so is_polytrope=false converts from CGS into geometric units, so the user would not want to use this for a PT EoS, even if they don't want to rescale.